lib/python/vdm/server/HTTPListener.py

```diff
@@ -1471,25 +1471,29 @@ def get(database_id):
                 try:
                     response = requests.get(url)
                 except Exception, err:
-                    return jsonify({'status': 'error', 'hostname': server['hostname']})
+                    return jsonify({"status": 200, "statusString": "OK", 'error': 'error', 'hostname': server['hostname']})
```

Comment:
Please don't add properties in the json response that are not specified in the ticket. statusString should indicate the error details, if any.